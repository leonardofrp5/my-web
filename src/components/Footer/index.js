import styles from './styles.module.scss'

function Footer () {
  const year = new Date().getFullYear()
  return <footer className={styles.footer}>Fabricated by Leonardo Romero Copyright Â© {year}</footer>
}

export default Footer
