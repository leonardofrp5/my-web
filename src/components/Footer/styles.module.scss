@import 'scss/variables/colors';

.footer {
  align-items: center;
  background-color: $malibu;
  bottom: 0;
  display: flex;
  font-size: 0.7em;
  height: 30px;
  justify-content: center;
  position: absolute;
  text-align: center;
  width: 100%;
}
