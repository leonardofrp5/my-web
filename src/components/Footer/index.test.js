import { render } from '@testing-library/react'

import Footer from './index'

describe('#Footer', () => {
  test('Check component Footer', () => {
    const { container } = render(<Footer />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
