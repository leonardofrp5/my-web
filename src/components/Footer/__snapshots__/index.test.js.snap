// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Footer Check component Footer 1`] = `
<footer
  class="footer"
>
  Fabricated by Leonardo Romero Copyright Â© 
  2021
</footer>
`;
